content/docs/esc/other-integrations/github-actions.md

```diff
@@ -0,0 +1,221 @@
+---
+title_tag: Integrate with GitHub Actions | Pulumi ESC
```

Comment:
There is more than one way to do this (like our new auth-actions and OIDC trust), as well as other integrations (like setting environment variables, not just using in Pulumi IaC.  Should we have multiple flavors of this doc?

Reply:
@cleverguy25  - the scope of this initial commit is to add docs to match the workshop that we're running next week. I suspect that we'll want to add sections for these alternative options but will defer to @interurban on how he wants to staff/deliver those additional docs.

Reply:
@cleverguy25 @interurban I ended up filing #12503 to track these suggestions.